### PR TITLE
Added navigation component

### PR DESCRIPTION
**Pull request by EgijaLocmele**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -10,34 +10,11 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (artikvatruwka) [inline public/index.html:12]: icon was removed from files, so remove it here also

File: public/manifest.json [hunk 2]
```diff
@@ -1,25 +1,13 @@
...
-  "name": "Create React App Sample",
+  "name": "Create React App Exercise",
   "icons": [
     {
       "src": "favicon.ico",
       "sizes": "64x64 32x32 24x24 16x16",
       "type": "image/x-icon"
-    },
-    {
-      "src": "logo192.png",
```
Comment (artikvatruwka) [inline public/manifest.json:11]: Why do you remove icons for different devices?

File: src/features/header/HeaderNavigation.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import { NeutralColors } from '@fluentui/theme';
+import { IconButton } from '@fluentui/react/lib/Button';
+
+export function HeaderNavigation() {
+  const shoppingCart={iconName: 'ShoppingCart'};
+  const hamburgerButton={iconName: 'GlobalNavButton'};
+  
+  return (
+    <header className="navigation" style={{ background: NeutralColors.gray60 }}>
+      <img className="navigation__logo" src={logo} alt="logo"></img>
```
Comment (artikvatruwka) [inline src/features/header/HeaderNavigation.js:12]: `` here

File: src/features/header/HeaderNavigation.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import logo from '../../assets/logo.svg';
+import { NeutralColors } from '@fluentui/theme';
+import { IconButton } from '@fluentui/react/lib/Button';
+
+export function HeaderNavigation() {
+  const shoppingCart={iconName: 'ShoppingCart'};
+  const hamburgerButton={iconName: 'GlobalNavButton'};
+  
+  return (
+    <header className="navigation" style={{ background: NeutralColors.gray60 }}>
```
Comment (artikvatruwka) [inline src/features/header/HeaderNavigation.js:11]: why inline styles? use either css or scss

File: src/features/header/HeaderNavigation.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+      <img className="navigation__logo" src={logo} alt="logo"></img>
+      <div className="navigation__buttons">
+        <IconButton iconProps={shoppingCart} className="navigation__buttons-style"/>
```
Comment (artikvatruwka) [inline src/features/header/HeaderNavigation.js:14]: `navigation__buttons-*style*` what stands behind style in this class, classes are made to be styled

File: src/App.js [hunk 6]
```diff
@@ -1,56 +1,13 @@
...
-import { Counter } from './features/counter/Counter';
-import './App.css';
+import { HeaderNavigation } from './features/header/HeaderNavigation';
+import { initializeIcons } from '@fluentui/react/lib/Icons';
+
+initializeIcons();
 
 function App() {
   return (
     <div className="App">
```
Comment (artikvatruwka) [inline src/App.js:9]: Is there any reason to have div wrapping everithing?

File: public/manifest.json (hunk 2 above)
Comment (EgijaLocmele) [reply public/manifest.json:11]: Those logos aren't used anymore and don't see a point of their usage here

File: src/features/header/HeaderNavigation.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
+import './HeaderNavigation.scss';
```
Comment (roman497) [inline src/features/header/HeaderNavigation.js:1]: wrong import order, should be components, logo, styles

**Review by roman497 — APPROVED** [review]

**Review by roman497 — APPROVED** [review]

**Review by artikvatruwka — APPROVED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

**Review by ddraycon — APPROVED** [review]

**Pull request merged**